schema/test/unit/computed_columns/project_revision_project_manager_form_changes_by_label_test.sql

```diff
@@ -1,7 +1,7 @@
 begin;
 SET client_min_messages TO WARNING; -- don't show all the truncate messages
 
-select plan(6);
+select plan(7);
```

Comment:
those tests are probably more complex than they should be now, but I guess that's fine